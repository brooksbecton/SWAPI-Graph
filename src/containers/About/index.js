import React from "react";

const About = () => {
  return (
    <div>
      <h1>About</h1>
      <h2>Photo Credits</h2>
      All photos from Rober Gonz√°lez at{" "}
      <a href="https://unsplash.com/photos/ShXbeohihGo">Unsplash</a>
    </div>
  );
};

export default About;
