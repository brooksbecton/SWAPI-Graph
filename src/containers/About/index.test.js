import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import About from "./index";

Enzyme.configure({ adapter: new Adapter() });

describe("About", () => {
  it("renders", () => {
    const tree = shallow(<About />);
    expect(tree).toMatchSnapshot();
  });
});
