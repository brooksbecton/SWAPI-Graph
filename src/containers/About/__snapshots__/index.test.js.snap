// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          About
</h1>,
        <h2>
          Photo Credits
</h2>,
        "All photos from Rober Gonz치lez at",
        " ",
        <a
          href="https://unsplash.com/photos/ShXbeohihGo"
>
          Unsplash
</a>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "About",
        },
        "ref": null,
        "rendered": "About",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Photo Credits",
        },
        "ref": null,
        "rendered": "Photo Credits",
        "type": "h2",
      },
      "All photos from Rober Gonz치lez at",
      " ",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Unsplash",
          "href": "https://unsplash.com/photos/ShXbeohihGo",
        },
        "ref": null,
        "rendered": "Unsplash",
        "type": "a",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            About
</h1>,
          <h2>
            Photo Credits
</h2>,
          "All photos from Rober Gonz치lez at",
          " ",
          <a
            href="https://unsplash.com/photos/ShXbeohihGo"
>
            Unsplash
</a>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "About",
          },
          "ref": null,
          "rendered": "About",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Photo Credits",
          },
          "ref": null,
          "rendered": "Photo Credits",
          "type": "h2",
        },
        "All photos from Rober Gonz치lez at",
        " ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Unsplash",
            "href": "https://unsplash.com/photos/ShXbeohihGo",
          },
          "ref": null,
          "rendered": "Unsplash",
          "type": "a",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
